### DISCUSSION TITLE
Building from master

### DISCUSSION THREAD

**Discussion by cristian-rivera** [Jyers/Marlin#889] [General Discussion]
Out of curiosity, how many people here are compiling from master and using that regularly instead of waiting for releases?

Anything worth noting besides the obvious of it being a WIP?

**Comment by Tuncay-Ayhan** [comment]
> Out of curiosity, how many people here are compiling from master and using that regularly instead of waiting for releases?
> 
> Anything worth noting?

I'm running my own fork from Jyers. With my own settings baked in. But parallel to his repo. So I have almost "daily" betas compiled for myself on my fork. You can check it out if you want. Relatively easy to setup and I don't have to compile it myself. "GitHub Actions" does that for me the way Jyers(Jacob) does it.

**Reply by cristian-rivera** [reply]
Awesome, thanks for the info, I'll certainly check this out!

**Comment by embraceentropy** [comment]
I'm also running my own fork from Jyers. I pull his updates and merge into my fork. I've edited some of the other code outside of the config files to my liking/needs.

**Comment by Tuncay-Ayhan** [comment]
1.3.3 incoming!



Op zo 23 mei 2021 om 16:49 schreef Cristian Rivera ***@***.***

**Reply by cristian-rivera** [reply]
Would you look at that timing 😂

**Comment by smooreace** [comment]
Im bouncing back and forth on testing settings based on the prior release
on 11 May, but also have available the auto-compiled firmware based on my
config files that Jyers offers for Patreons. That firmware is auto compiled
every hour based on his most recent work.
